packages/agent/conf/demos/triall/oid4vci_metadata/triall.json

```diff
@@ -56,40 +56,40 @@
           }
         ],
         "order": [
-          "Voornaam",
-          "Achternaam",
+          "firstName",
+          "lastName",
           "email",
           "type"
         ],
         "credentialSubject": {
-          "Voornaam": {
+          "firstName": {
             "value_type": "string",
             "display": [
               {
-                "name": "FirstName"
+                "name": "First Name"
               },
               {
-                "name": "FirstName",
+                "name": "First Name",
                 "locale": "en-US"
               },
               {
-                "name": "Voornaam",
+                "name": "firstName",
                 "locale": "nl-NL"
               }
             ]
           },
-          "Achternaam": {
+          "lastName": {
             "value_type": "string",
             "display": [
               {
-                "name": "LastName"
+                "name": "Last Name"
```

Comment:
check locale

Reply:
Good question, when you connect with a browser on Japanese or French, should it display in Dutch? I do not know, doesn't seem logical to me.